doc/whats_new.rst

```diff
@@ -408,10 +411,10 @@ People who contributed to this release (in alphabetical order):
 .. _Mostafa Khalil: https://github.com/mkhalil8
 .. _Nick Tolley: https://github.com/ntolley
 .. _Orsolya Beatrix Kolozsvari: http://github.com/orbekolo/
+.. _Rajat Partani: https://github.com/raj1701
```

Comment:
we'll update this later ... this list is for the previous release, don't touch that

Reply:
oh never mind looks like you just put it in the right place alphabetically